Runtime/Data/Operation/Extraction/ValueExtractor.cs

```diff
@@ -109,5 +109,21 @@ protected virtual void OnDisable()
         {
             Result = default;
         }
+
+        /// <summary>
+        /// Attempts to invoke the <see cref="Extracted"/> event if the <see cref="Result"/> is not null.
+        /// </summary>
+        /// <param name="data">The data to emit.</param>
+        /// <returns>Whether the event was invoked.</returns>
+        protected virtual bool InvokeEvent(TEventElement data)
+        {
+            if (Result == null)
```

Comment:
```suggestion
            if (data == null)
```
Isn't this suggested change what is intended?

Reply:
the `data` maybe of a type that is not nullable (e.g. `Vector3`) but the `Result` will be of nullable type.

The reason for this is, the type can be nullable but the event payload can't be nullable because Unity can't display a nullable field like its non nullable counterpart

So a UnityEvent with Vector3 payload shows in the inspector properly, but a Vector3? payload doesn't show up in the inspector as you'd expect.

Reply:
👍